### ISSUE TITLE
Error when installing

### ISSUE THREAD

**Issue by leocastroz** [SteveYuOWO/vue-hot-toast#6]
Can anyone help me install the library, I'm having problems ?

```
$: npm i @steveyuowo/vue-hot-toast --save
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:..
```

**Comment by SteveYuOWO** [comment]
@leocastroz Sorry for the late reply

https://github.com/SteveYuOWO/vue-hot-toast/commit/969cbe626ceb20d516622a69ee551d72959972b4#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L29

This bug was fixed in v1.2.0

**Comment by leocastroz** [comment]
👋 Thanks @SteveYuOWO , I'll test it soon.
